under-review/sdfobject-operational_state.sdf.json

```diff
@@ -0,0 +1,56 @@
+{
+  "info": {
+    "copyright": "copyright 2016-2017, 2019 Open Connectivity Foundation, Inc. All rights reserved.",
+    "license": "https://github.com/one-data-model/oneDM/blob/master/LICENSE",
```

Comment:
Should we use "BSD-3-Clause"?

Reply:
SDF:

```
   The license string is preferably either a URI that points to a web
   page with an unambiguous definition of the license, or an [SPDX]
   license identifier.  (For models to be handled by the One Data Model
   liaison group, this will typically be "BSD-3-Clause".)
```

I'd prefer SPDX if there is one.

Reply:
+1 if there is a SPDX identifier, then that should be used, else use URL to license page